### ISSUE TITLE
Export to PDF and select certain page range does not uncheck all pages option

### ISSUE THREAD

**Issue by jacovandermolen** [trevorsandy/lpub3d#105]
### Subject
Export to PDF and select certain page range does not uncheck all pages

### Environment
* Version of LPub3D - 2.3.0 Revision 0 Build 887 Hash g1b66997

### Steps to reproduce
Export to PDF and provide a certain page range renders all pages.

### Expected behaviour
It should render and export the pages I want.

### Actual behavior
Export to PDF > select Page Range > type ie. 1-2 > OK starts rendering all pages.
This is because the All pages option is not unchecked.

### Workaround
Manually uncheck All pages

**Comment by trevorsandy** [comment]
Yep - I broke something fiddling around with the page range form for the Dark Mode theme. This should be corrected in v2.3.1. I've already done the work. 

FYI, you'll also notice the page range dialogue does not allow you to use continuous previous page processing (going from the end to beginning of the document). This has also been corrected in the 2.3.1.

Cheers,

**Comment by jacovandermolen** [comment]
Great! Looking forward to 2.3.1 :-)
